bin/nib

```diff
@@ -126,25 +155,22 @@ if __name__ == '__main__':
             print('Done')
 
         elif options.action == 'wizard':
-            empty = True
-            for dirpath, dirs, files in os.walk('.'):
-                if len(dirs) > 0 or len(files) > 0:
-                    empty = False
-                    break
-
-            overwrite = input('This directory is not empty, the wizard will overwrite existing files. Proceed? [y/N]: ')
-            if not overwrite.lower().startswith('y'):
-                sys.exit(1)
+            # If the directory's not empty
+            if os.listdir('.') != []:
+                answer = getYN( 'This directory is not empty, '
+                                'the wizard will overwrite existing files.'
+                                ' Proceed? ', False)
```

Comment:
Fixed the space.  The code goes way over 80 chars so I just tagged the space after "files. ", and fixed the hanging indent.